### ISSUE TITLE
Gas Cloud spawns in Zangarmarsh appear incorrect/low spawn rate

### ISSUE THREAD

**Issue by wakeland15** [ValorenWoW/sunstrider-core#5418]
Did a full loop. Only saw three spawns (yes, I have goggles on) and one of the spawns was in a place I've never seen on a TBC server or Classic (which was very close to 2.4.3 spawns). While I cannot be sure I was the only one farming at the time, I asked several time in general and whispered all high levels to see if there was just heavy competition and no one else seemed to be farming at that time. I have not checked Nag/Nether/SMV yet, but will get to those tonight.

**Comment by wakeland15** [comment]
Okay I did some further testing. Spawn points are definitely a little off, especially in Netherstorm. Really that's not an issue, spawns are spawns. My best guess on the problem is this: Only a small amount of clouds are spawned at any given time. And I was definitely the only person doing the farm, I left an easy bait cloud up in Netherstorm and it was there every time I rode by. Not going to tell you how to run your server, but my suggestion is increasing the amount of clouds present on the map at any given time. I like the scripting that a new cloud insta spawns somewhere when one is sucked, however. Got lucky and had one spawn right on top of me after I sucked the first one.
